cmd/utils/flags.go

```diff
@@ -378,6 +417,11 @@ var (
 		Name:  "miner.extradata",
 		Usage: "Block extra data set by the miner (default = client version)",
 	}
+	// MinerGasPriceFlag = BigFlag{
```

Comment:
why is this commmented? should we remove?

Reply:
removed. It's a legacy flag for us.